backend/typescript/models/camper.model.ts

```diff
@@ -9,8 +9,8 @@ export interface Camper extends Document {
   allergies: string;
   hasCamera: boolean;
   hasLaptop: boolean;
-  earlyDropoff: string;
-  latePickup: string;
+  earlyDropoff: string[];
+  latePickup: string[];
```

Comment:
sorry I should have clarified, these should be Date types in the model itself but just strings in the DTOs D: